scheduler/pkg/kafka/topics.go

```diff
@@ -50,51 +50,54 @@ func NewTopicNamer(namespace string, topicPrefix string) *TopicNamer {
 	}
 	return &TopicNamer{
 		namespace:   namespace,
-		topicPrefix: topicPrefix,
+		topicPrefix: fmt.Sprintf("%s.", topicPrefix),
```

Comment:
why are we adding a dot here? and removing the dot from string formatter in `GetModelErrorTopic` etc.? easier to construct a regex?

Reply:
Likewise find this a bit confusing, as I'd expect to see the delimiters all together for clarity rather than having one over here

Reply:
We want to allow any prefix and then add a dot which is our separator. At present, don't check if their prefix ends in a dot as doesn't really matter.